Packs/Darktrace/ReleaseNotes/2_0_6.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Darktrace Model Breach Comment Count
+- Fixed display name of incident field incident_darktracemodelbreachcommentcount
```

Comment:
This is not right, work with the SDK command to generate the right message

Reply:
fixed it now